static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -337,6 +337,16 @@ define([
     }
 
     function initEndSlate(player, endSlatePath) {
+        var fader = $('.video__fader');
```

Comment:
Why is this in the `initEndSlate`?
I think I can see why this is done, but given the complexity of this file already, it'd be good to shift it else where.